concepts/0011-decorators/README.md

```diff
@@ -0,0 +1,256 @@
+  may be undesirable. If community members want to explore this, they are encouraged
+  to discuss broadly, first.)
+* **Connection**. It is conceivable that Alice could bundle up a set of decorators
+  and send them to Bob as a way of saying "Hey, Bob: unless/until subsequent interactions
+  give you more specific semantics, here's a set of semantics (settings for our connection)
+  that you should assume for all interactions." (This may or may not be useful. It introduces ambiguity--should settings declared
+  at this scope override those at the scope of a message family or message type? It
+  also adds a burden of remembering to another party; some of what informs the interaction
+  isn't present in the bits that flow later on. Thus, this style of decorator is also
+  left as a possibility for future exploration and is not actively supported at present.)
+
+## Reference
+
+This section of this RFC will be kept up-to-date with a list of globally accepted
+decorators, and links to the HIPEs that define them.
+
+* [`~thread`](../0008/message-id-and-threading): provide request/reply and threading semantics
+* [`~timing`](https://github.com/hyperledger/indy-hipe/pull/68): timestamps, expiration, elapsed time
+* [`~trace`](https://github.com/hyperledger/indy-hipe/pull/60): collaborative debugging and monitoring
+* [`~l10n`](https://github.com/hyperledger/indy-hipe/pull/64): localization support
+ 
+
+## Drawbacks
+
+By having fields that are meaningful yet not declared in core schemas, we run the risk that
+parsing and validation routines will fail to enforce details that are significant but invisible.
+We also accept the possibility that interop may look good on paper, but fail due to
+different understandings of important metadata.
+
+We believe this risk will take care of itself, for the most part, as real-life usage
+accumulates and decorators become a familiar and central part of the thinking for
+developers who work with agent-to-agent communication.
+
+# Rationale and alternatives
+[alternatives]: #alternatives
+
+There is ongoing work in the `#indy-semantics` channel on Rocket.Chat to explore the concept
+of __overlays__. These are layers of additional meaning that accumulate above a __schema
+base__. Decorators as described here are quite similar in intent. There are some subtle
+differences, though. The most interesting is that decorators as described here may be
+applied to things that are not schema-like (e.g., to a message family as a whole, or to
+a connection, not just to an individual message).
+
+We may be able to resolve these two worldviews, such that decorators are viewed as overlays
+and inherit some overlay goodness as a result. However, it is unlikely that decorators
+will change significantly in form or substance as a result. We thus believe the current
+mental model is already HIPE-worthy, and represents a reasonable foundation for immediate
```

Comment:
Can you change this to RFC?

Reply:
Good catch. Fixed. I also fixed a couple other places that I hadn't caught.